io/net/virt-net/lpm.py

```diff
@@ -56,14 +78,8 @@ def setUp(self):
         self.hmc_pwd = self.params.get("hmc_pwd", '*', default='********')
         self.options = self.params.get("options", default='')
         self.net_device_type = self.params.get("net_device_type", default='')
-        self.lpar = self.get_partition_name("Partition Name")
-        if not self.lpar:
-            self.cancel("LPAR Name not got from lparstat command")
-        self.lpar_ip = self.get_mcp_component("MNName")
-        if not self.lpar_ip:
-            self.cancel("LPAR IP not got from lsrsrc command")
         self.session = Session(self.hmc_ip, user=self.hmc_user,
-                               password=self.hmc_pwd)
+                                password=self.hmc_pwd)
```

Comment:
get rid of this space, they run a bot that checks the formatting of the code and it'll complain about it